app/src/libs/combat/ai_v2.ts

```
@@ -215,10 +215,34 @@ export const performAIaction = (
+          case "round_greater_than":
+            return nextBattle.round > condition.value;
+          case "round_lower_than":
+            return nextBattle.round < condition.value;
           case "does_not_have_summon":
             return !nextBattle.usersState.find(
               (u) => u.controllerId === user.userId && u.isSummon && u.curHealth > 0,
             );
+          case "has_effect":
+            const userEffects = nextBattle.usersEffects.filter(
+              (e) => e.targetId === user.userId && e.type === condition.effectType,
+            );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)